src/Microsoft.TestPlatform.PlatformAbstractions/netcore/Tracing/EqtTrace.cs

```diff
@@ -0,0 +1,380 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.IO;
+    using System.Threading;
+
+    /// <summary>
```

Comment:
We can move documentation for public members to `IPlatformEqtTrace` and use `/// <inheritdoc/>` to reduce duplication.